notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        plt.plot(x, normal_dist, color="blue", linestyle="solid", label="Gaussian")
</pre><p>Always use <code>plt.figure()</code> before staring new figure, otherwise while saving figures one figure will go in others. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>